library/ctr_drbg.c

```diff
@@ -90,15 +90,13 @@ int mbedtls_ctr_drbg_seed_entropy_len(
     /*
      * Initialize with an empty key
      */
-    if( ( ret = mbedtls_aes_setkey_enc( &ctx->aes_ctx, key, MBEDTLS_CTR_DRBG_KEYBITS ) ) != 0 )
-    {
+    if( ( ret = mbedtls_aes_setkey_enc( &ctx->aes_ctx,
+                                        key, MBEDTLS_CTR_DRBG_KEYBITS ) ) != 0 )
         return( ret );
-    }
 
     if( ( ret = mbedtls_ctr_drbg_reseed( ctx, custom, len ) ) != 0 )
-    {
```

Comment:
Why are we removing brackets here? I know we don't often have them for single statements, but a lot of coding standards require them as defensive programming. Can you put them back in please?